shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -45,11 +45,11 @@ - (void)layoutSubviews {
   if ([self.layer isKindOfClass:[CAEAGLLayer class]]) {
     CAEAGLLayer* layer = reinterpret_cast<CAEAGLLayer*>(self.layer);
     layer.allowsGroupOpacity = YES;
-    layer.opaque = YES;
     CGFloat screenScale = [UIScreen mainScreen].scale;
     layer.contentsScale = screenScale;
     layer.rasterizationScale = screenScale;
   }
+  self.layer.opaque = [self flutterViewController].viewIsOpaque;
```

Comment:
That's a good point.  the way it's implemented right now, changing this wouldn't dynamically change the `FlutterView` until a relayout happened (e.g. changing screen orientation or resizing the view).

I think if we went the route you're suggesting, I'd rather have this be an initializer parameter on `FlutterViewController` as well though, as the property form of it would become even more confusing in that context.  That said, I think having it as an initializer parameter might make more sense anyway.

Reply:
Trying to figure out if/how any of this impacts storyboard usage.

Reply:
Another option (which might allow for more natural iOS programming) is to have an initializer parameter that says whether the FlutterView automatically sets this at all.

Reply:
I'm now passing the `FlutterViewController` as a WeakPtr into the `FlutterView` initializer.  I've also wired the property so we can do this dynamically, as @matthew-carroll pointed out we might want to do this when initially loading the view to avoid a flash of content.